data/sql/updates/pending_db_world/rev_1729603880657213700.sql

```diff
@@ -0,0 +1,60 @@
+--
+INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES (1549, 4155.55, 6962.69, 164.671, 3.40091, 571, 'Coldarra');
+INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES (1548, 3954.58, 6045.14, 23.8309, 3.93498, 571, 'TheWestrift');
+INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES (1547, 3580.99, 6654.6, 195.569, 6.0948, 571, 'TransitusShield');
```

Comment:
```suggestion
INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES
(1549, 4155.55, 6962.69, 164.671, 3.40091, 571, 'Coldarra'),
(1548, 3954.58, 6045.14, 23.8309, 3.93498, 571, 'TheWestrift'),
(1547, 3580.99, 6654.6, 195.569, 6.0948, 571, 'TransitusShield'),
```
Etc. for the rest of the entries. Also, what is the reason for two separate files?

Reply:
I thought it would be the most correct action per zone, I can make them the same file if that's the proper way to do it

Reply:
Should it be like this?

![image](https://github.com/user-attachments/assets/151b4f60-399b-4598-88b1-9a40bc58d310)

Reply:
No, there should be only a single line with:
```sql
INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES
```
The rest should just be the data values.

Reply:
> I thought it would be the most correct action per zone, I can make them the same file if that's the proper way to do it

I would personally have them be the same file but separate them with a comment, e.g.
```sql
-- Borean Tundra
INSERT (the rest)
-- Sholazar Basin
INSERT (the rest)
```

Reply:
![image](https://github.com/user-attachments/assets/43a85ff0-2472-4416-8b12-de80472ca1ca)

LIke this?

Reply:
Yes, but the `DELETE` statement should actually include all the IDs that you are inserting to ensure that the query is rerunnable. Also, instead of a semicolon at the end of every line for the data, it should be commas except for the last line.

Reply:
When I get all the format the correct way I will test again on a empty database as i did with the previous queries.

This is the correct way?

![image](https://github.com/user-attachments/assets/97cdf970-e4e0-497e-b570-8506bf8529c4)

Reply:
> This is the correct way?

Yes.

Reply:
> Yes.
All my time I'm using REPLACE instead of DELETE+INSERT
Why I'm wrong in my choise?